lbz/exceptions.py

```diff
@@ -216,6 +216,13 @@ class FailedDependency(LambdaFWClientException):
     status_code = HTTPStatus.FAILED_DEPENDENCY.value
 
 
+class TooEarly(LambdaFWClientException):
```

Comment:
You should also add this new Exception to the test module, to verify it the same way as others 😉 

In the future, it would be good to make it more automatic because it is simple to forget about it.

Reply:
I actually thought its already automatic.

Reply:
That's how you implemented it 🤷